src/Maths/Silk.NET.Intrinsics/Avx/AvxRegister.Float.cs

```diff
@@ -0,0 +1,330 @@
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> DotProduct3(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> DotProduct4(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Add(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Subtract(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Equal(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> NotEqual(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Negate(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Lerp(WorkUnit<float> left, WorkUnit<float> right, WorkUnit<float> amount)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Slerp(WorkUnit<float> left, WorkUnit<float> right, WorkUnit<float> amount)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Concatenate(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Multiply(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Transform2(WorkUnit<float> value, WorkUnit<float> quaternion)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Transform3(WorkUnit<float> value, WorkUnit<float> quaternion)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Transform4(WorkUnit<float> value, WorkUnit<float> quaternion)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Abs(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> HorizontalAdd(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Shuffle(WorkUnit<float> value, byte control)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> X(WorkUnit<float> vector)
```

Comment:
Is this `GetX`?

Reply:
Yes.

Reply:
Is there meant to be a `SetX` or `WithX` counterpart? (depending on if mutable or immutable)